
try {
    const aList = Android.testFunction()
    console.log(aList);
} catch(e) {
    console.log("Error: ", e)
}